### PR TITLE
remove n_proc

### PR DESCRIPTION
**Pull request by BitcoinMechanic**

### CODE REVIEW

**Review by dr-bonez — CHANGES_REQUESTED** [review]
Remove `${N_PROC}` in `make -j${N_PROC}` on line 129

**Review by dr-bonez — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -128,7 +126,7 @@ RUN ./configure LDFLAGS=-L`ls -d /opt/db*`/lib/ CPPFLAGS=-I`ls -d /opt/db*`/incl
   --with-utils \
   --with-libs \
   --with-daemon
-RUN make -j${N_PROC}
+RUN make -j
```
Comment (chrisguida) [inline Dockerfile:129]: does `-j` work like this? just... with no core count specifier?

Comment (BitcoinMechanic) [reply Dockerfile:129]: yes

Comment (chrisguida) [reply Dockerfile:129]: why not just remove `-j` entirely?

**Pull request merged**